.github/workflows/doc-build.yml

```diff
@@ -4,9 +4,14 @@ on:
   pull_request:
   merge_group:
   push:
-    branches:
-      - master
-      - develop
+    tags:
+      # Match all release tags including beta, rc
+      - '[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+'
+      - '[0-9]+.[0-9]+.beta[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+.beta[0-9]+'
+      - '[0-9]+.[0-9]+.rc[0-9]+'
+      - '[0-9]+.[0-9]+.[0-9]+.rc[0-9]+'
   workflow_dispatch:
```

Comment:
Why this verbosity?

Reply:
copied like this from dist.yml; I agree that it may be overly specific